.github/workflows/ex_doc_version_check.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: ex_doc_version_check
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight
+
+
+jobs:
+  test:
+    name: Check ex_doc version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get current version
+        run: |
+            CURR=`curl -s https://api.github.com/repos/elixir-lang/ex_doc/tags | \
+                  grep "\"name\":" | \
+                  sort --reverse | \
+                  head -n 1 | \
+                  awk '{print substr($2, 2, length($2)-3)}'`
+            TITLE="\`ex_doc\` ${CURR} released"
+            ISSUE_EXISTS=`curl -s https://api.github.com/repos/starbelly/rebar3_ex_doc/issues | grep -c "\"title\": \"${TITLE}\""`
+
+            if [ ${ISSUE_EXISTS} -ne 0 ]
+            then
+              RESP=`curl -L \
+                -X POST \
+                https://api.github.com/repos/starbelly/rebar3_ex_doc/issues \
+                -d '{"title":"${TITLE}","body":"This is an automated report."}'`
```

Comment:
I wonder if this won't run in all forks, too (?)

Is there a way to further restrict this workflow? Otherwise forks either have to disable it by hand or have their "Actions" tab with issues every now and then (I'm assuming they can't just post an issue as simply as this, but have to use some kind of token).

Reply:
Yes, I believe it would. Prolly some env variable with current repo name exist, so we can wrap the whole task into IF clause.

Reply:
Or even better, not run the workflow in forks at all, otherwise it still pollutes the forks' tabs (though it might not be important 🤷).

Reply:
It seems this changed recently, and now actions are opt-in, so I'll go ahead and resolve this one 👍 

<img width="737" alt="image" src="https://github.com/starbelly/rebar3_ex_doc/assets/7303768/df496286-4d42-4f6f-a164-5ffcdffb3042">